html.md

```diff
@@ -64,23 +64,25 @@
 ## 3. Buttons
   > Buttons can submit a form, be a link styled like a button, or simply just a button.
 
+Buttons should always have the `type` attribute set to prevent possible bugs from the default `type=submit`
```

Comment:
I think "to prevent possible bugs" is a little too vague and doesn't really hit on the reason for specifying this as a standard. We need to convey that buttons placed inside forms that are not meant to submit the form should have a specific `type='button'` set to prevent the default submit action from the default `type='submit'`.